engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -429,6 +430,16 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
     }
   }
 
+  private def validateWdlFiles(workflowInputs: WorkflowCoercedInputs): ErrorOr[WorkflowCoercedInputs] = {
+    val failedFiles = workflowInputs.collect {
+      case (callName, WdlSingleFile(value)) if value.startsWith("\"gs://") => "Invalid value for File input '" + callName + "':"+ value + " starts with a '\"' "
```

Comment:
ToL: I find callName to be slightly misleading, it seems like it's the fullyQualifiedName for the input.